Python/ceval.c

```diff
@@ -3380,6 +3381,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
                 STAT_INC(LOAD_ATTR, deferred);
                 cache->adaptive.counter--;
                 oparg = cache->adaptive.original_oparg;
+                STAT_DEC(LOAD_ATTR, unquickened);
```

Comment:
We don't need this in LOAD_GLOBAL as well?

Reply:
These counters are uints. Is there no danger of underflow?

Reply:
Yes, thanks

Reply:
> These counters are uints. Is there no danger of underflow?

Unsigned int underflow is well defined in C (unlike for signed int). So the STAT_DEC followed by the STAT_INC is a no op for all values.